cmd/touch/touch.go

```diff
@@ -31,6 +31,7 @@ func init() {
 	flags.BoolVarP(cmdFlags, &notCreateNewFile, "no-create", "C", false, "Do not create the file if it does not exist.")
 	flags.StringVarP(cmdFlags, &timeAsArgument, "timestamp", "t", "", "Use specified time instead of the current time of day.")
 	flags.BoolVarP(cmdFlags, &localTime, "localtime", "", false, "Use localtime for timestamp, not UTC.")
+	flags.BoolVarP(cmdFlags, &recurse, "recursive", "R", false, "Recurse into the listing.")
```

Comment:
You need to declare variable `recurse` above, same place as `localTime`.

Reply:
@albertony Ah thank you for that, I have done that now.